vita3k/gui/src/game_context_menu.cpp

```diff
@@ -52,9 +63,15 @@ void delete_game(GuiState &gui, HostState &host, const std::string &title_id) {
     }
 }
 
+static auto delete_game_popup = false;
+static auto delete_savedata_popup = false;
+static auto check_savedata_and_shaderlog = false;
+static auto copy_save_data_popup = false;
+
 void game_context_menu(GuiState &gui, HostState &host, bool *selected, const std::string &title_id) {
-    const fs::path save_data_path{ fs::path(host.pref_path) / "ux0/user/00/savedata" };
-    const fs::path shaderlog_path{ fs::path(host.base_path) / "shaderlog" };
+    const auto save_data_path{ fs::path(host.pref_path) / "ux0/user" / host.io.user_id / "savedata" / title_id };
+    const auto copy_save_data_path{ fs::path(host.pref_path) / "ux0/user" / fmt::format("{:0>2d}", copy_save_data_id) / "savedata" / title_id };
+    static const auto shaderlog_path{ fs::path(host.base_path) / "shaderlog" };
```

Comment:
```suggestion
    const auto shaderlog_path{ fs::path(host.base_path) / "shaderlog" };
```

Reply:
No, static good here, base path is permanant path, no can change

Reply:
There is no benefit. Only use static in a global context.

Reply:
i keep it, use static for permanent path, never can change is good

Reply:
No advantage really. And if base_path changes for whatever reason in the future this will cause problems and bug fixing will be a pain.